test_scripts/Polices/appID_Management/TS01.lua

```diff
@@ -0,0 +1,83 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary: 
```

Comment:
Please follow the structure of template: first is "Description:" after that " Requirement summary:"
https://github.com/CustomSDL/sdl_atf_test_scripts/pull/32/commits/f717d13cb14018dff5e1f362e290c4f16eabc55a